### PR TITLE
Create NASA and ESA directories for DPS

### PR DESCRIPTION
**Pull request by nemo794**
This PR reorganizes the `get-dem` repo; it adds specific ESA and NASA directories for each team to store their JNs, DPS packaging code, etc.

The NASA-specific files have be moved into the `nasa` directory.

Inside the `esa` directory is a dummy file to establish the directory structure; this file can be deleted once the ESA code is added to the repo.

Please tag or add anyone I missed. (For example, is Nathan from ESA able to review?)

Thank you!

### CODE REVIEW

**Comment by sujen1412** [comment]
You will need to make some adjustments to the run script to support this change. 
https://github.com/nemo794/get-dem/blob/esa_nasa_directory_structure/nasa/run.sh#L28

```
basedir=$( cd "$(dirname "$0")" ; pwd -P )
```
is the path to the run script, so the path to the get_dem script is one more level above. 
So 
```
REPO_ROOT_PATH=$(dirname ${basedir})
python ${REPO_ROOT_PATH}/get_dem.py
```

**Review by wildintellect — DISMISSED** [review]

File: nasa/run.sh
Comment (sujen1412) [inline nasa/run.sh]: Input and output paths are relative to the working directory. 
So use something like 
```
mkdir -p ${PWD}/output
```

**Review by sujen1412 — CHANGES_REQUESTED** [review]

File: nasa/run.sh
Comment (nemo794) [reply nasa/run.sh]: Ah, is this documented anywhere? I looked in the official MAAP documentation, but could not find the method for assigning the location of the output directory. Then I looked at the gdal Hello World example, and it simply used `output` ([link](https://repo.maap-project.org/sshah/gdal-hello-world/-/blob/main/run.sh)).

To confirm -- does this mean that the best practice for any location of the `run.sh` script is to set the output directory to `${PWD}/output`?

Related -- Is there a way to please note in the documentation the convention of how/where to place the `output` dir?

Thank you!

Comment (nemo794) [reply nasa/run.sh]: Hi @sujen1412 , I just tested and had a successful DPS run using simply `mkdir -p output`.

Should we leave it simple here, or add in `${PWD}/` ?

Thank you again!

Comment (sujen1412) [reply nasa/run.sh]: `${PWD}/` is optional but makes it explicit.

**Review by sujen1412 — APPROVED** [review]

**Comment by chuckwondo** [comment]
@nemo794, any reason not to merge this? Are you waiting for @esamaap to approve before merging?

**Comment by nemo794** [comment]
Hi @chuckwondo , Yep -- I would like for ESA MAAP folks to review it first, to double-check that we're all on the same page and have correct Github accesses setup.

**Review by ghost — APPROVED** [review]

**Review by maapEsaCap — APPROVED** [review]

**Pull request merged**
